src/Models/MonitoredScheduledTask.php

```diff
@@ -94,6 +94,7 @@ public function markAsFinished(ScheduledTaskFinished $event): self
             'runtime' => $event->task->runInBackground ? 0 : $event->runtime,
             'exit_code' => $event->task->exitCode,
             'memory' => $event->task->runInBackground ? 0 : memory_get_usage(true),
+            'output' => $this->getEventTaskOutput($event),
```

Comment:
I'm thinking that some commands could potentially generate a lot of output that you don't want to store in the db. I'm thinking we should add a config option `store_output_by_default` that is set to `true`. If people don't want to store output, they can set it to `false`.

We could also add methods `storeOutput()`, and `doNotStoreOutput` that can be put on the task, that override the default.

Reply:
Ok, so instead of you having to opt out from storing the output to db, I decided to just add another method `storeOutputToDb()`. 
Now only if you call this method the output will be stored in db. This ensure it's bc regardless of what you called before.

Reply:
Nice! So, I guess this is now ready for a final review?

Reply:
Yes. Also if you can take a look at my [comment](https://github.com/spatie/laravel-schedule-monitor/pull/43#discussion_r630017762).